azimuth_caas_operator/utils/ansible_runner.py

```diff
@@ -2,20 +2,19 @@
 import json
 import logging
 import os
-import yaml
 
-from cryptography.hazmat.primitives.asymmetric import ed25519
+import yaml
 from cryptography.hazmat.primitives import serialization
-
+from cryptography.hazmat.primitives.asymmetric import ed25519
 from easykube import ApiError
 
 from azimuth_caas_operator.models.v1alpha1 import cluster as cluster_crd
-from azimuth_caas_operator.models.v1alpha1 import cluster_type as cluster_type_crd
+from azimuth_caas_operator.models.v1alpha1 import \
+    cluster_type as cluster_type_crd
 from azimuth_caas_operator.utils import cluster_type as cluster_type_utils
 from azimuth_caas_operator.utils import image as image_utils
 from azimuth_caas_operator.utils import k8s
 
-
```

Comment:
I think the space changes are causing problems here. I think these need reverting please.

Reply:
Should be done in 63f25f6